packages/beyonic/src/Adaptor.js

```diff
@@ -1,9 +1,7 @@
-import {
-  execute as commonExecute,
-  expandReferences,
-} from '@openfn/language-common';
+import { execute as commonExecute } from '@openfn/language-common';
```

Comment:
Yeah, just to clarify, looking at the code to beyonic there is no public change at all.